src/core/server/extensions/playerFunctions/inventory.ts

```diff
@@ -21,12 +21,12 @@ function getFreeInventorySlot(p: alt.Player, tabNumber: number = null): { tab: n
         const tab = p.data.inventory[i];
 
         // Go to next tab if inventory is full.
-        if (tab.length >= 27) {
+        if (tab.length >= 28) {
```

Comment:
Maybe you can create a constant for the value `28 `e.g `INVENTORY_MAX_SIZE `to make it more self explaning if someone reads the code in the future.